python/apps/utils/rabbitmq.py

```diff
@@ -84,3 +84,73 @@ def call(self, queryset=None, serializer=None, exchange_name='', exchange_type='
         print(" [.] Got %r" % self.response)
 
         return self.response
+
+    def on_response(self, ch, method, properties, body):
+        print('I have got the messsage from server')
+        self.response = body
+
+
+
+    def call_java(self, queryset=None, serializer=None, exchange_name='', exchange_type='topic', q_receiving='',
+             q_sending='', message=''):
+
+        self.response = None
+        self.q_sending = q_sending
+        self.q_receiving = q_receiving
```

Comment:
You are assigning to objects variables, but I did not see any usage of these objects variables. E.g. in other class methods.

Reply:
it is used in queue binding statement.